frontend/src/metabase/databases/components/DatabaseEngineField/DatabaseEngineWidget.styled.tsx

```diff
@@ -37,7 +37,8 @@ export const EngineCardRoot = styled.li<EngineCardRootProps>`
   border-radius: 0.375rem;
   background-color: ${color("white")};
   cursor: pointer;
-  outline: ${props => (props.isActive ? `2px solid ${color("focus")}` : "")};
+  outline: ${props =>
+    props.isActive ? `2px solid var(--mb-color-focus)` : ""};
```

Comment:
(Wondering if we could shorten this with `&&`, if not then it's fine. Also, this would evaluate to `outline: ;` if the isActive props is falsey.)